### PR TITLE
Eshields have 100% block vs energy ,but 0 vs everything else

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**

### CODE REVIEW

File: code/game/objects/items/weapons/shields.dm [hunk 1]
```diff
@@ -54,8 +54,6 @@
 	var/active = 0
 
 /obj/item/weapon/shield/energy/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance)
-	if(active)
```
Comment (Jalleo) [inline code/game/objects/items/weapons/shields.dm]: Umm this a mistake or ya removing it from working?

Comment (KorPhaeron) [inline code/game/objects/items/weapons/shields.dm]: It doesn't work at blocking things.

But it works at reflecting things.

**Comment by Xhuis** [comment]
This is dumb. Energy shields are already too expensive to be viable, there isn't a reason to further piledrive them into the ground. :-1:

File: code/game/objects/items/weapons/shields.dm (hunk 1 above)
Comment (Jalleo) [inline code/game/objects/items/weapons/shields.dm]: Yep just read rest of code and worked it out

**Comment by KorPhaeron** [comment]
This is important because of another change I am making please hold

**Comment by Xhuis** [comment]
Then it might be a good idea to put that in the OP instead of having no description whatsoever.

**Comment by Zilenan91** [comment]
Yeah currently this is just a nerf for no reason on an item that can already be shat on by hard counters.

**Comment by KorPhaeron** [comment]
>50% blockrate against bullets and 50% damage reduction from bullets on the nuke op armour
>hard counters

Uh huh

**Comment by Zilenan91** [comment]
stunbaton/prod

**Comment by duncathan** [comment]
@Zilenan91 he has another PR open that lets shields block stun batons

**Comment by Zilenan91** [comment]
I saw it

**Comment by WJohn** [comment]
I'm okay with this.

**Comment by Niknakflak** [comment]
I always saw them as like the halo shields the jackels had which did in fact block bullets.  I'd rather just keep them the way they were.

**Comment by AlexanderUlanH** [comment]
I'm okay with this as long as the eshield gets 20 throwforce while active

**Comment by Cheridan** [comment]
wouldn't this make actual riot shields better than energy shields

**Comment by KorPhaeron** [comment]
>wouldn't this make actual riot shields better than energy shields

Depends on what weapons the enemy is using.

It's still a 100% chance to reflect lasers/tasers back at the enemy, which is significantly stronger than the riot shield.

**Comment by tkdrg** [comment]
Sure, we can try this out, with the baton thing. I think it's for the best.

**Comment by Lumipharon** [comment]
>This is important because of another change I am making please hold
>merges it anyway

k.

**Comment by tkdrg** [comment]
@Lumipharon the other change was the stun baton change that kor closed in favor of xxalpha's and that I merged at the same time.

**Comment by Lumipharon** [comment]
What, but that literally has nothing to do with this change, considering this pr makes e-shield have 0 chance to block batons at all?

**Comment by tkdrg** [comment]
This PR has nothing to do with batons, Lumi. The end result of both PRs is that eshields have 100% vs energy, 100% vs batons, and 0% vs bullets.

**Comment by Lumipharon** [comment]
Ok well that's just very confusing wording then, since this pr implies that only energy weapons (batons having never been included with them) are blocked by eshields, and nothing else.

**Comment by tkdrg** [comment]
That's what this PR does, true. The baton thing is another PR entirely that I merged at the same time like Kor requested, and I linked above (#13823).

**Comment by Jordie0608** [comment]
>no changelog

**Pull request merged**